drivers/input/Kconfig

```diff
@@ -34,4 +34,11 @@ source "drivers/input/Kconfig.xpt2046"
 
 endmenu # Input Drivers
 
+config INPUT_MULTITOUCH_ENABLED
+    bool "Enable multitouch"
```

Comment:
you have few compliance checks around these ones, fyi you can run it locally with `./scripts/ci/check_compliance.py`

Reply:
Yes, my bad, I forgot that time. Fixed. :)